code/src/components/Card.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react'
+
```

Comment:
I don't understand why there are spaces between the imports. But I don't think it wrong.